superset/assets/src/types/react-table.d.ts

```diff
@@ -0,0 +1,224 @@
+// Type definitions for react-table 7
```

Comment:
the newest version of `react-table` doesn't have types in https://github.com/DefinitelyTyped/DefinitelyTyped

found these online and adjusted to for the features we're using.